pkg/rpc/client/policy.go

```diff
@@ -0,0 +1,87 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/native"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/pkg/errors"
+)
+
+// PolicyContractHash represents BE hash of native Policy contract.
+var PolicyContractHash = util.Uint160{154, 97, 164, 110, 236, 151, 184, 147, 6, 215, 206, 129, 241, 91, 70, 32, 145, 208, 9, 50}
+
+// GetMaxTransactionsPerBlock invokes `getMaxTransactionsPerBlock` method on a
+// native Policy contract.
+func (c *Client) GetMaxTransactionsPerBlock() (int64, error) {
+	result, err := c.InvokeFunction(PolicyContractHash.StringLE(), "getMaxTransactionsPerBlock", []smartcontract.Parameter{}, nil)
+	if err != nil {
+		return 0, err
+	} else if result.State != "HALT" || len(result.Stack) == 0 {
+		return 0, errors.New("invalid VM state")
+	}
+
+	return topIntFromStack(result.Stack)
+}
+
+// GetMaxBlockSize invokes `getMaxBlockSize` method on a native Policy contract.
+func (c *Client) GetMaxBlockSize() (int64, error) {
+	result, err := c.InvokeFunction(PolicyContractHash.StringLE(), "getMaxBlockSize", []smartcontract.Parameter{}, nil)
+	if err != nil {
+		return 0, err
+	} else if result.State != "HALT" || len(result.Stack) == 0 {
+		return 0, errors.New("invalid VM state")
+	}
+
+	return topIntFromStack(result.Stack)
+}
+
+// GetFeePerByte invokes `getFeePerByte` method on a native Policy contract.
+func (c *Client) GetFeePerByte() (int64, error) {
+	result, err := c.InvokeFunction(PolicyContractHash.StringLE(), "getFeePerByte", []smartcontract.Parameter{}, nil)
+	if err != nil {
+		return 0, err
+	} else if result.State != "HALT" || len(result.Stack) == 0 {
+		return 0, errors.New("invalid VM state")
+	}
+
+	return topIntFromStack(result.Stack)
```

Comment:
I guess the top three functions can be generalized with one internal function parameterized by `functionToCall`.